internal/pkg/workspace/workspace.go

```diff
@@ -317,6 +382,16 @@ func IsInGitRepository(fs FileStat) bool {
 	return !os.IsNotExist(err)
 }
 
+// PipelineManifestLegacyPath returns the path to pipeline manifests before multiple pipelines (and the copilot/pipelines/ dir) were enabled.
+func (ws *Workspace) PipelineManifestLegacyPath() (string, error) {
+	copilotPath, err := ws.copilotDirPath()
+	if err != nil {
+		return "", err
+	}
+	pipelineManifestPath := filepath.Join(copilotPath, pipelineFileName)
+	return pipelineManifestPath, nil
```

Comment:
nit: same change as below:
```suggestion
	return filepath.Join(copilotPath, pipelineFileName), nil
```